doc/src/devenv/tipsandtricks.rst

```diff
@@ -110,3 +110,22 @@ Check the coding style of PHP code
   vm$> cd /var/www/lc/site
   vm$> ./vendor/bin/phpcs --runtime-set ignore_warnings_on_exit true -s \
     && echo You can commit: No errors found!
+
+Algolia indices configuration
+---------------------------
+For configuring algolia in development environment first specify the Apllication ID and Admin API Key in ``parameter.yml``
+Clear the indices and import the settings from algolia
```

Comment:
You say here it imports the settings, below you say it imports the indices. What is correct?